chapters/14-team-package-dev.Rmd

```diff
@@ -1,19 +1,53 @@
 # R package development in a team-based environment
 
-- Making use of GitHub Organizations/Teams
-- Branch protection, member roles
-- Resolving conflict, personality differences
-- Running meetings, code of conduct
-- Decision making
-- Governance
-- Issue assignment
-- Relying on and using code of conduct to build culture and standards
-- Setting up SSH, two-factor authentication, PATs?
+-   Making use of GitHub Organizations/Teams
+-   Branch protection, member roles
+-   Resolving conflict, personality differences
+-   Running meetings, code of conduct
+-   Decision making
+-   Governance
+-   Issue assignment
+-   Relying on and using code of conduct to build culture and standards
+-   Setting up SSH, two-factor authentication, PATs?
 
+## Final exercises
 
+<!-- NOTE: Preferably something linked to working towards completing the project assignments, maybe to be used as a milestone? -->
 
-## Final exercise
+### For the first project assignment
 
-<!-- NOTE: Preferably something linked to working towards completing the project assignments, maybe to be used as a milestone? -->
+These exercises are to be done on the *first project assignment* (the
+individual-based one). You won't be able to practice everything that you've
+learned during this chapter for this exercise.
+
+1. Set up branch protection for your main branch in your package repository on
+GitHub.
+2. Create a file called `GOVERNANCE.md` (FIXME: Should this be here?) and write
+some general guidelines on:
+    - How decisions are made.
+    - If and how meetings will be run and held.
+    - What culture and standards are valued and how they will be shaped.
+    - How conflict/disagreements will be resolved.
+3. On your GitHub account, set up 2FA and SSH.
+4. In your GitHub package repository settings, create a PAT for working with...
+TODO: This item might already be done for setting up GitHub actions.
+
+### For the third project assignment
 
+These exercises are to be done for the *third, team-based project assignment*.
 
+1. Assign one team member to create a GitHub Organization account. 
+Have this team member add the other members to the Organization.
+    - Set each team member with "Maintainer" permissions.
+2. Assign another team member to create a new repository where the
+project package will be hosted.
+    - This team member will then set up branch protection of the main
+    branch.
+3. In the new repository, create a new file called `GOVERNANCE.md`.
```

Comment:
For the team based governance section, I think we should also suggest a team-working activity to create this contract. I wrote this perevioously:

> We recommend that generate the team work contract according to these guidelines
>
>- For 5 minutes, each team member will silently write out 4 different suggestions for the team work contract on their computer.
>- Next, paste all suggestions into a common Google doc (or similar)
  and arrange them to put similar suggestions together.
>- Spend 10 -15 minutes discussing the suggestions
  and decide which ones you will use for your team contract either by consensus or voting
  (you can of course add new ones at this point if any were missed earlier)
>- Add the final contract in a file called `team-contract.md` in the repo root
  (in organizations this file might be kept internal instead of committed to a public repo).
>    - These are the principles you adhere to when working.
>    - This file should only be updated if there is agreement between team members to do so.

Reply:
Added as comment in text for now.